src/app/__snapshots__/index.test.js.snap

```diff
@@ -61,42 +61,7 @@ THE SOFTWARE IS PROVIDED \\"AS IS\\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR I
 "
 `;
 
-exports[`generator:app README.md contains plugin name 1`] = `
```

Comment:
this test would use my stored creds, so I made it check the stuff we're looking for specifically